### PR TITLE
Send email when status order change

### PR DESCRIPTION
**Pull request by ghost**
## Related Tickets
- [#TicketID](https://edu-redmine.sun-asterisk.vn/issues/43421)

## WHAT (optional)
- Change status orders, send email to customer when has change status

## HOW
- Add new def **approve**, **reject**, **cancel**, **update_quantity_reject**, **send_mail_change_status**  in orders controller
- Add new view mailer 
- Change i18n in file locales

## Evidence (Screenshot or Video)
image

### CODE REVIEW

**Comment by thanghd-1196** [comment]
ready

**Comment by ghost** [comment]
ready

File: app/controllers/admin/orders_controller.rb [hunk 1]
```diff
@@ -10,6 +10,41 @@ def show
     @total = total_money @order
   end
 
+  def approve
```
Comment (ghost) [inline app/controllers/admin/orders_controller.rb:13]: Các trạng thái này đều bị lặp code rất nhiều em làm gọn lại nhé

**Review by ghost — COMMENTED** [review]
cmt

**Comment by thanghd-1196** [comment]
ready

**Comment by ghost** [comment]
ready

File: config/routes.rb [hunk 2]
```diff
@@ -2,6 +2,9 @@
   namespace :admin do
     resources :orders, except: %i(create destroy)
     resources :products
+    get "approve/:id", to: "orders#approve", as: :approve_order
+    get "reject/:id", to: "orders#reject", as: :reject_order
+    get "cancel/:id", to: "orders#cancel", as: :cancel_order
```
Comment (vanvtt-0952) [inline config/routes.rb]: sao ko dùng `member ` nhỉ

**Comment by ghost** [comment]
ready

**Pull request merged**